client/src/components/new-request/search.vue

```diff
@@ -343,6 +343,7 @@ export default class NewSearch extends Vue {
     newReqModule.mutateIsPersonsName(value)
   }
   get location () {
+    if (['MVE'].includes(this.request_action_cd)) newReqModule.mutateLocation('BC')
```

Comment:
Could you please add code comments explaining things here?

Reply:
Definitely!